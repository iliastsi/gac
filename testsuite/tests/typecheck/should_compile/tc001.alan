hello () : proc
	foo : int;
{
	foo = 2147483648;  -- overflow
	foo = -2147483648; -- no overflow
}
