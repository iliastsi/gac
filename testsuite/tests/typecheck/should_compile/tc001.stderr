tc001.alan:4:8: Warning: Overflow in implicit constant conversion at `2147483648'
	Integer has been rounded to `-2147483648'


